cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -1301,49 +1301,37 @@ func (snr *sovereignNodeRunner) CreateManagedProcessComponents(
 	requestedItemsHandler := cache.NewTimeCache(
 		time.Duration(uint64(time.Millisecond) * coreComponents.GenesisNodesSetup().GetRoundDuration()))
 
-	extraHeaderSigVerifierHolder := headerCheck.NewExtraHeaderSigVerifierHolder()
-	sovHeaderSigVerifier, err := headerCheck.NewSovereignHeaderSigVerifier(cryptoComponents.BlockSigner())
-	if err != nil {
-		return nil, err
-	}
-
-	err = extraHeaderSigVerifierHolder.RegisterExtraHeaderSigVerifier(sovHeaderSigVerifier)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
This whole code should've happen inside `sovereignRunTypeComp` -> `Create`

Reply:
To minimize the conflicts with the next PRs, I created another PR for this at the end of refactor 
https://github.com/multiversx/mx-chain-go/pull/6140